### PR TITLE
[feature] - adds airport + googleFlights API 

### PR DESCRIPTION
**Pull request by AtlasCoughed**
[feature] - adds airport + googleFlights API 
[feature] - connects with receiveFile

### CODE REVIEW

File: client/src/actions/getLatLng.js [hunk 1]
```diff
@@ -37,6 +37,15 @@ export function getLatLng() {
         .catch((error) => {
           console.error('Error inside axios get request for geocode address:', error);
         });
+		    // findAirport(lat, long);
+      var data = {
+		    latitude: lat,
+        longitude: long,
+      };
+      dispatch({
+        type: 'LAT_LONG',
```
Comment (JonathanCMitchell) [inline client/src/actions/getLatLng.js]: import as const inside actionTypes

File: client/src/containers/UserInput.jsx [hunk 2]
```diff
@@ -1,4 +1,4 @@
-import React, { Component } from 'react';
```
Comment (JonathanCMitchell) [inline client/src/containers/UserInput.jsx]: use previous React, { Component } syntax

File: server/model/googleFlights.model.js [hunk 3]
```diff
@@ -4,42 +4,77 @@ const request = require('request');
 const userFlights = module.exports;
```
Comment (JonathanCMitchell) [inline server/model/googleFlights.model.js]: const Promise = require('bluebird');

File: client/src/containers/UserInput.jsx [hunk 4]
```diff
@@ -158,8 +155,9 @@ class UserInput extends Component {
               placeholder="$400"
               min="150"
               max="99999999999"
-              validate={true}
-              required>
+              validate
```
Comment (JonathanCMitchell) [inline client/src/containers/UserInput.jsx]: why is validate={true} missing? Did this resolve the propTypes error?

Comment (AtlasCoughed) [inline client/src/containers/UserInput.jsx]: Yes resolves error

**Pull request merged**